js/app/modules/contentGroup/mediaLightbox.js

```diff
@@ -57,6 +57,8 @@ const MediaLightbox = new Module.Class({
     },
 
     _on_close: function () {
+        this.lightbox_widget.destroy();
```

Comment:
I don't think you should call `destroy()` manually, since it will destroy the widget in C while still keeping a dangling JS object. Instead, `this.lightbox_widget = null` should be enough, since the setter in `Lightbox.Lightbox` will remove the widget from its container and the widget will get destroyed at the next garbage collection if nothing else is holding on to it.